datafusion/src/physical_plan/analyze.rs

```diff
@@ -206,4 +208,9 @@ impl ExecutionPlan for AnalyzeExec {
             }
         }
     }
+
+    fn statistics(&self) -> Statistics {
+        // TODO stats: validate that we don't need to provide statistics on an EXPLAIN plan
```

Comment:
I do not think stats on an explain plan add much/any value

Reply:
removed TODO in that sense